awx/main/tasks.py

```diff
@@ -314,7 +314,7 @@ def send_notifications(notification_list, job_id=None):
         update_fields = ['status', 'notifications_sent']
         try:
             sent = notification.notification_template.send(notification.subject, notification.body)
-            notification.status = "successful"
+            notification.status = "successful" or "running"
```

Comment:
Notifications have a `status`, and it represents whether or not sending the notification "worked".  They only have a few states:

![image](https://user-images.githubusercontent.com/214912/58731523-ec1e2680-83bc-11e9-8bc1-21c639cb61d3.png)

I think this change is unnecessary.

Reply:
> notification.status = "successful" or "running"

Am I missing something here, or isn't this just always going to assign `"successful"` anyway?  Since `s1 or s2` will only evaluate to `s2` in the case where `s1` is empty, and `"successful"` isn't empty.

Reply:
@ghjm 

Nope.  Syntactically, this will *always* evaluate to `notification.status = "successful"`.

This code should just be removed.  I think it was just copypasta on @beeankha's part.

Reply:
This was a case of `"running" status` = 🔨 and any place with that logic (for notifications) looking like a nail.  My strategy was to overdo it a bit and then prune back redundancy as needed.  Will remove this code!